server/chat/backend/agent/tools/victorops_tool.py

```
@@ -0,0 +1,110 @@
+            "success": True,
+            "count": len(incidents),
+            "results": incidents,
+        })
+    except Exception as exc:
+        logger.exception("[VICTOROPS-TOOL] get_incidents failed for user=%s", user_id)
+        return json.dumps({"error": f"Error fetching incidents: {str(exc)[:200]}"})
+
+
+def get_victorops_teams(
+    user_id: Optional[str] = None,
+    **kwargs,
```

Ruff (0.15.12)

[warning] Missing type annotation for `**kwargs`

(ANN003)